src/qt/forms/debugwindow.ui

```diff
@@ -1198,6 +1198,9 @@
                  <property name="textFormat">
                   <enum>Qt::PlainText</enum>
                  </property>
+                 <property name="wordWrap">
+                  <bool>true</bool>
+                 </property>
```

Comment:
Is this still needed if we hardcode it to use one line per flag?

Reply:
I think, @jonatack is right, and this change seems redundant.

Reply:
This was referring to the force new line version.
Since we are concluding that forcing a new line is not preferred we should leave it.